src/config.rs

```diff
@@ -187,22 +185,42 @@ impl Default for TermvizConfig {
     }
 }
 
-pub fn get_config() -> Result<TermvizConfig, confy::ConfyError> {
+pub fn ask_store() -> bool {
+    let mut x = String::with_capacity(5);
+    print!("Store default config? (y|n): ");
+    let _ = io::stdout().flush();
+    io::stdin().read_line(&mut x).expect("Error reading input");
+    let x = x.strip_suffix("\n").unwrap();
+    match x {
```

Comment:
convert input to lower case first!

Reply:
oops, i had that at some point. will fix